sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/DataFeedLiveTests.cs

```diff
@@ -487,11 +488,12 @@ public async Task CreateAndGetSqlServerDataFeedWithOptionalMembers()
         }
 
         [RecordedTest]
-        [TestCase(AzureBlobDataFeedSource.AuthenticationType.Basic)]
-        [TestCase(AzureBlobDataFeedSource.AuthenticationType.ManagedIdentity)]
-        public async Task CreateAndGetAzureBlobDataFeedWithAuthentication(AzureBlobDataFeedSource.AuthenticationType authentication)
+        [TestCase("Basic")]
+        [TestCase("ManagedIdentity")]
+        public async Task CreateAndGetAzureBlobDataFeedWithAuthentication(string authenticationType)
         {
             MetricsAdvisorAdministrationClient adminClient = GetMetricsAdvisorAdministrationClient();
+            var authentication = GetAuthenticationInstance<AzureBlobDataFeedSource.AuthenticationType>(authenticationType);
```

Comment:
Unfortunately we can't use the types as arguments directly anymore, so we're passing a string and converting it with a helper method.